code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -1,5 +1,21 @@
 #define DEFAULT_DOOMSDAY_TIMER 4500
 
+GLOBAL_LIST_INIT(blacklisted_malf_machines, typecacheof(list(
+		/obj/machinery/field/containment,
+		/obj/machinery/power/supermatter_shard,
+		/obj/machinery/dominator,
```

Comment:
Guess that one is skirting a little too close to balancing. Will remove it for now, though leaving the ability for the AI to detonate a round-ending device at whim does seem like a major oversight/exploit to me. Likewise is the ability to bring it to life and have it attack friendly gang members who are defending it.